frontend/src/app/components/page-without-sidebar/page-without-sidebar.component.sass

```diff
@@ -1,5 +1,10 @@
-tasque-header
+.container
   width: 100%
-  height: 8%
-main
-  padding-top: 5%
+  height: 100%
+
+.header
+  min-width: 1110px
```

Comment:
Does it mean, that _header_ will not scale less than _1110px_? Or i misunderstood?

Reply:
If the value is less a horizontal scroll appears below. It is necessary that the elements do not overlap each other. It may be changed in the future

Reply:
This way is better to hide some elements (for example, make one menu point with all other inside it) or reduce other elements width
But, as we are going to use high resolution monitors, we can try this for now

Reply:
I will keep in mind)